packages/database/README.md

```diff
@@ -1,44 +1,5 @@
 # @firebase/database
 
-This is the realtime database component for the Firebase JS SDK. It has a peer
-dependency on the [`@firebase/app`](https://npm.im/@firebase/app) package on NPM. This package
-is included by default in the [`firebase`](https://npm.im/firebase) wrapper
-package.
+This is the realtime database component for the Firebase JS SDK. 
```

Comment:
Nit: I am also leaning towards "This is the X component **of** the Firebase JS SDK" instead of "This is the X component for the Firebase JS SDK".